chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx

```diff
@@ -15,31 +15,37 @@ import useChatContextStore from "../../hooks/useChatContextStore";
 import useWindowDimensions from "../../hooks/useWindowDimensions";
 import { errorUILoadingPaneStyleProps } from "./common/errorUIStyleProps/errorUILoadingPaneStyleProps";
 
-export const LoadingPaneStateful = (props: ILoadingPaneProps) => {
+// export const LoadingPaneStateful = (props: ILoadingPaneProps, startChatErrorPaneProps: IStartChatErrorPaneProps) => {
+// export const LoadingPaneStateful = (props: ILoadingPaneStatefulParams) => {
+export const LoadingPaneStateful = (props: any) => {
+    console.log("ADAD LoadingPaneProps props", props);
     const [state, ]: [ILiveChatWidgetContext, Dispatch<ILiveChatWidgetAction>] = useChatContextStore();
-    const generalStyleProps: IStyle = Object.assign({}, defaultGeneralLoadingPaneStyleProps, props.styleProps?.generalStyleProps);
-    const styleProps: ILoadingPaneStyleProps = {
-        ...props.styleProps,
-        generalStyleProps: generalStyleProps
+    const { loadingPaneProps, startChatErrorPaneProps } = props;
+    console.log("ADAD loadingPaneProps", loadingPaneProps);
+    console.log("ADAD startChatErrorPaneProps", startChatErrorPaneProps);
+    const generalLoadingPaneStyleProps: IStyle = Object.assign({}, defaultGeneralLoadingPaneStyleProps, loadingPaneProps.styleProps?.generalStyleProps);
+    const loadingPaneStyleProps: ILoadingPaneStyleProps = {
+        ...loadingPaneProps.styleProps,
+        generalStyleProps: generalLoadingPaneStyleProps
     };
     const errorUIStyleProps: ILoadingPaneStyleProps = {
         ...errorUILoadingPaneStyleProps
     };
 
-    const controlProps: ILoadingPaneControlProps = {
+    const loadingPaneControlProps: ILoadingPaneControlProps = {
         id: "oc-lcw-loading-pane",
         dir: state.domainStates.globalDir,
-        ...props.controlProps
+        ...loadingPaneProps.controlProps
     };
 
     const errorUIControlProps: ILoadingPaneControlProps = {
+        ...props.controlProps,
         id: "oc-lcw-alert-pane",
         dir: state.domainStates.globalDir,
-        titleText: "Chat is failing to load.",
-        subtitleText: "Please Close the chat and try again.",
+        titleText: startChatErrorPaneProps.controlProps?.titleText ?? "Chat is failing to load.",
```

Comment:
Let's use the one we're already using in 1033.js. Don't introduce new strings here

Reply:
updating strings